src/sage/benchmark/__init__.py

```diff
@@ -0,0 +1,128 @@
+from sage.all import *
```

Comment:
FWIW: in #35049 (trivial PR moving some existing benchmark files to the same place) I used `sage.tests.benchmarks` instead of `sage.benchmark`. No opinion on which is best, but I guess we don't want to keep both.